release-notes-1.3.0.md

```diff
@@ -1,8 +1,7 @@
-!not-ready-for-release!
-
 #### Version Number
 ${version-number}
 
 #### New Features
+- Updated to base the image off the [opensuse-tini-image](https://github.com/CAFapi/opensuse-tini-image)
```

Comment:
This is really just an internal re-organisation.  It doesn't have any effect on a consumer so I wouldn't update the release notes at all.  We will not release 1.3 just for this.

Reply:
Reverted release notes change.